app/stac_api/management/commands/populate_testdb.py

```diff
@@ -0,0 +1,168 @@
+import os
+import logging
+import json
+import pprint
+
+from django.contrib.gis.geos import GEOSGeometry
+
+from stac_api.models import Asset
+from stac_api.models import Collection
+from stac_api.models import CollectionLink
+from stac_api.models import Item
+from stac_api.models import ItemLink
+from stac_api.models import Keyword
+from stac_api.models import Provider
+from django.core.management.base import BaseCommand
+from django.conf import settings
+
+# from django.core.management.base import CommandError
+
+# path definiton relative to the directory that contains manage.py
+DATADIR = settings.BASE_DIR / 'app/stac_api/management/sample_data/'
+logger = logging.getLogger(__name__)
+
+
+def create_provider():
+    pass
+
+def create_collection_link(collection, link):
+    # Hier noch Fehler abfangen Index und Key, darauf achten, dass einige Properties
+    # allenfalls nicht vorhanden sind und das auch ok ist, weil sie optional sind.
```

Comment:
I would keep all comment and code in english

Reply:
Yes, you are right. This particular german text is my fault, sorry. I just wrote that very quickly before pushing (more meant as information to @boecklic) on my day off, so that Chris could continue working on that.

Reply:
already removed, thanks for passing this info, made my life easier ;-)